### PR TITLE
Do you know what happens at a Sprint Planning meeting? - update content

### PR DESCRIPTION
**Pull request by vladislav-kir**
As per my conversation with @adamcogan, need to mention these two points in "Do you know what happens at a Sprint Planning meeting?" rule:
1. Tech Improvements
2. Tech Debt

Also replaced the the image for "Just Enough Refactoring" rule, because the last one did not work

### CODE REVIEW

File: rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md [hunk 1]
```diff
@@ -28,6 +30,10 @@ The Product Owner proposes how the product could increase its value and utility
 
 What can be delivered in the Increment resulting from the upcoming Sprint?
 
+[The Architecture Review](/do-you-conduct-an-architecture-review-after-every-sprint) should be conducted every few sprints prior to the Sprint Planning to determine any tech improvements that can be implemented in the project.
+
+[The Technical Debt](/technical-debt) should be considered prior to the Sprint Planning, as it is imperative to address it. [Just Enough Refactoring](/just-enough-refactoring) should be used to solve the debt that directly surrounds or is impacted by the features included into the sprint.
```
Comment (JackDevAU) [inline rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md:35]:
```suggestion
[Technical Debt](/technical-debt) should be considered prior to the Sprint Planning, as it is imperative to address it. [Just Enough Refactoring](/just-enough-refactoring) should be used to solve the debt that directly surrounds or is impacted by the features included into the sprint.
```

File: rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ authors:
     url: https://ssw.com.au/people/uly
 related:
   - do-you-create-a-sprint-forecast-aka-the-functionality-that-will-be-developed-during-the-sprint
+  - do-you-conduct-an-architecture-review-after-every-sprint
+  - technical-debt
```
Comment (JackDevAU) [inline rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md:11]: should we add `just-enough-refactoring` as a related rule as well?

File: rules/just-enough-refactoring/rule.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ In a large project, you add a feature without taking the time to clean up any of
 Conversely, in a large project, you decide to add a feature and take on the herculean task of trying to fix all the existing technical debt at once. While this might feel like a responsible approach, it's akin to "boiling the ocean" - a task so huge that it's likely to stall progress and overwhelm your team. This approach can lead to significant project delays and could also introduce new bugs as you touch parts of the system that aren't directly related to the new feature.
 
 ::: bad 
-![Figure: Bad example - Boiling the Ocean](boil-the-ocean.webp)
+![Figure: Bad example - Boiling the Ocean](boil-the-ocean.png)
```
Comment (JackDevAU) [inline rules/just-enough-refactoring/rule.md:27]: 🚀

File: rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md (hunk 2 above)
Comment (vladislav-kir) [reply rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md:11]: Done :)

File: rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md (hunk 1 above)
Comment (vladislav-kir) [reply rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md]: Done :)

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**